DearPyGui/src/core/AppItems/composite/mvAboutWindow.hpp

```diff
@@ -0,0 +1,29 @@
+// to be sure to have the best speed from compiler
+// without having to check the behavior of each compiler
+// is better to use both pragma once and guards
+// in that way can get the best of both (cross-platform and help compilation speed)
+
+#ifndef mvAboutWindow_hpp           // guards
+#define mvAboutWindow_hpp           // guards
```

Comment:
I would like to do this BUT I would like to do it in a single suite.

I'd also like to keep the ".h" extension for now.